src/components/App/App.js

```diff
@@ -66,7 +70,14 @@ const App = () => {
           <Route exact path="/countdown" render={() => <CountdownPane etd={user.attributes.etd} err={error} dob={user.attributes.date_of_birth}/>}/>
           <Route exact path="/obituary" render={() => <ObitPane obit={user.attributes.obituary}/>}/>
           <Route exact path="/executors" render={() => <ExecutorPane executor={user.attributes.executor}/>}/>
-          <Route exact path="/photoadd" render={() => <PhotoAdd updateProfilePicture={updateProfilePicture} currentProfilePic={user.attributes.profile_picture_url}/>}/>
+          <Route exact path="/add-photo/profile" render={() => <PhotoAdd updateProfilePicture={updateProfilePicture} currentProfilePic={user.attributes.profile_picture_url} type={"profile"}/>}/>
```

Comment:
These paths refer to paths you're defining within the FE app? To redirect to a given component?

Reply:
yep! This is how we simulate the kind of navigation a user is familiar with i.e. with the URL changing. It allows them to send the URLs to friends etc too!

Reply:
Sorry did not realize the submit button was taking the user to the dashboard